module/metrics.go

```diff
@@ -191,12 +191,18 @@ type VerificationMetrics interface {
 	// It increases the total number of result approvals.
 	OnResultApproval()
 
-	// LogVerifiableChunkSize is called whenever a verifiable chunk is shaped for a specific
-	// chunk. It adds the size of the verifiable chunk to the histogram. A verifiable chunk is assumed
-	// to capture all the resources needed to verify a chunk.
-	// The purpose of this function is to track the overall chunk resources size on disk.
-	// Todo wire this up to do monitoring (3183)
-	LogVerifiableChunkSize(size float64)
+	// OnFinalizedBlockReceived is called whenever a finalized block arrives at the assigner engine.
+	// It increments the total number of finalized blocks.
+	OnFinalizedBlockReceived()
```

Comment:
Would a block height (gauge) be more useful than a counter here? Then we could compare to the finalized height metric of the follower and more easily see a delay (`consensus_compliance_finalized_height-verification_assigner_finalized_height`)

Reply:
Refactored in https://github.com/onflow/flow-go/pull/466/commits/ab5d68537c918e6d1261d625c79a29ffc21747e9.